pulseapi/entries/views.py

```diff
@@ -126,11 +128,19 @@ class EntryView(RetrieveAPIView):
     """
     A view to retrieve individual entries
     """
-    queryset = Entry.objects.public()
+
+    #queryset = Entry.objects.public()
+
+    def get_queryset(self):
```

Comment:
good point. Short answer: yes - if this should be "only approved" then we'll have to update the test for this because staff tests assume "seeing all entries"